### ISSUE TITLE
example error

### ISSUE THREAD

**Issue by mohammadSHP** [deepinsight/insightface#2106]
hello. I wanna try the example in Readme but i got this error. can somebody give me a advice?

`Applied providers: ['CPUExecutionProvider'], with options: {'CPUExecutionProvider': {}}
find model: /home/mohammad/.insightface/models/buffalo_l/1k3d68.onnx landmark_3d_68 ['None', 3, 192, 192] 0.0 1.0
Applied providers: ['CPUExecutionProvider'], with options: {'CPUExecutionProvider': {}}
find model: /home/mohammad/.insightface/models/buffalo_l/2d106det.onnx landmark_2d_106 ['None', 3, 192, 192] 0.0 1.0
Applied providers: ['CPUExecutionProvider'], with options: {'CPUExecutionProvider': {}}
find model: /home/mohammad/.insightface/models/buffalo_l/det_10g.onnx detection [1, 3, '?', '?'] 127.5 128.0
Applied providers: ['CPUExecutionProvider'], with options: {'CPUExecutionProvider': {}}
find model: /home/mohammad/.insightface/models/buffalo_l/genderage.onnx genderage ['None', 3, 96, 96] 0.0 1.0
Applied providers: ['CPUExecutionProvider'], with options: {'CPUExecutionProvider': {}}
find model: /home/mohammad/.insightface/models/buffalo_l/w600k_r50.onnx recognition ['None', 3, 112, 112] 127.5 127.5
set det-size: (640, 640)
/home/mohammad/Project/nextaibox/insightface_master/python-package/insightface/utils/transform.py:68: FutureWarning: `rcond` parameter will change to the default of machine precision times ``max(M, N)`` where M and N are the input matrix dimensions.
To use the future default and silence this warning we advise to pass `rcond=None`, to keep using the old, explicitly pass `rcond=-1`.
  P = np.linalg.lstsq(X_homo, Y)[0].T # Affine matrix. 3 x 4`

what should I do?!

**Comment by nttstar** [comment]
I can not see any error, it's just a warning.

**Comment by mohammadSHP** [comment]
> I can not see any error, it's just a warning.

yes you are right. but it does not detect faces and creat t1_output.jpg. it did not to anything.

**Comment by ArtemKiyashko** [comment]
Could maintainers apply single-word fix?
Here: insightface/utils/transform.py:68
Replace:
`P = np.linalg.lstsq(X_homo, Y)[0].T # Affine matrix. 3 x 4`

With:
`P = np.linalg.lstsq(X_homo, Y, rcond=None)[0].T # Affine matrix. 3 x 4`

Reference doc: https://numpy.org/doc/stable/reference/generated/numpy.linalg.lstsq.html

**Comment by ZubairKhan001** [comment]
@ArtemKiyashko its just a warning but it is occurring due to the face detector models especially buffalo_l and antelopev2 output bounding boxes overflow the maximum and minimum values. kindly check it. I mentioned this issue in #2365.

**Comment by changchiyou** [comment]
Any new about this issue? It's extremely annoy 😢 

![image](https://github.com/deepinsight/insightface/assets/46549482/bc5bdd24-7070-4bea-ba38-502ee0abe6b5)
